### PR TITLE
qa : fixes

### PR DESCRIPTION
**Pull request by thelostone-mc**
- refactoring
- added common nav
- added custom nav for grants
- new: removed min + hours + any
- new: increase textarea height + make form neater
- new : increased font-size
- detail: remove duplicate fund text
- milestones: change order + remove description
- milestones: UI cleanup
- milestones: add back button
- details: tab fixes (justify text + ensure post activity appears only on activity tab)

refs: #2798
refs: #2753
refs: #2803

untitled

### CODE REVIEW

File: app/grants/templates/grants/new.html [hunk 1]
```diff
@@ -67,65 +68,60 @@
         <div id="robot_error" class="hidden">
           <img src="{% static "v2/images/prime.png" %}">
         </div>
-        <div class="col-sm-12 col-md-10 col-lg-6" id="grants_form">
+        <div class="col-sm-12 col-md-10 col-lg-7" id="grants_form">
           <div class="form__intro text-center">
             <h1>{% trans "Create a Grant" %}</h1>
-            <p>{% trans "We're excited to learn about your project. Complete the form below to get funding for your project." %}</p>
+            <p>{% trans "We're excited to learn about your project.<br>Complete the form below to get funding for your project." %}</p>
```
Comment (mbeacom) [inline app/grants/templates/grants/new.html]: Mind splitting this off into separate `trans` tags with `<br>` in between them?

File: app/grants/templates/grants/quickstart.html [hunk 2]
```diff
@@ -103,8 +104,8 @@ <h2 class="font-title">{%trans "It’s not just about the money" %}</h2>
                   <img src="{% static "v2/images/grants/quickstart/fund.svg" %}">
                 </div>
                 <div class="col-12 text-center font-subheader">
-                  <h3 class="font-subheader mb-2">{%trans "Ready to fund a grant?" %}</h3>
-                  <a class="btn btn-gc-blue" href="/">{%trans "Fund a Grant" %}</a>
+                  <h3 class="font-subheader my-3">{%trans "Ready to fund a grant?" %}</h3>
+                  <a class="btn btn-gc-blue" href="{% url 'grants:grants' %}">{%trans "Fund a Grant" %}</a>
```
Comment (mbeacom) [inline app/grants/templates/grants/quickstart.html]: 🙌

File: app/grants/views.py [hunk 3]
```diff
@@ -56,15 +56,13 @@ def grants(request):
     if request.method == 'POST':
         keyword = request.POST.get('search_grants', '')
         _grants = Grant.objects.filter(
-            Q(description__icontains=keyword) |
-            Q(title__icontains=keyword) |
-            Q(reference_url__icontains=keyword)
+            Q(description__icontains=keyword) | Q(title__icontains=keyword) | Q(reference_url__icontains=keyword)
```
Comment (mbeacom) [inline app/grants/views.py]: _rabble rabble_

File: app/grants/views.py [hunk 4]
```diff
@@ -129,18 +127,20 @@ def grant_details(request, grant_id):
...
+               {
+                   'title': 'Fix the build by derp-nation',
+                   'date': '08.02.2018',
+                   'description': 'Initial commit with some blah blah blah...',
+               },
+               {
+                   'title': 'A subpar commit by derp-diggity',
+                   'date': '08.02.2018',
+                   'description': 'Initial commit with some blah blah blah...',
+               }]
```
Comment (mbeacom) [inline app/grants/views.py]: this autoformatting needs fixed

File: app/grants/templates/grants/new.html (hunk 1 above)
Comment (thelostone-mc) [reply app/grants/templates/grants/new.html]: _rabble rabble_

**Comment by thelostone-mc** [comment]
@mbeacom address the comments ^_^

**Pull request merged**